src/components/App/AppModal/index.vue

```diff
@@ -30,6 +30,8 @@ import NotificationsModal from "@/views/DashboardView/StudentDashboard/SubjectSe
 import ReferralModal from "@/views/DashboardView/StudentDashboard/ReferralModal";
 import SessionFulfilledModal from "@/views/SessionView/SessionFulfilledModal";
 import VolunteerOnboardingModal from "@/views/DashboardView/VolunteerDashboard/VolunteerOnboardingModal.vue";
+import ReferencesModal from "@/views/DashboardView/VolunteerDashboard/ReferencesModal.vue";
+import PhotoUploadModal from "@/views/DashboardView/VolunteerDashboard/PhotoUploadModal.vue";
```

Comment:
We can remove these imports since we're using a different modal system to show the references modal and photo upload modal